src/EnergyPlus/SingleDuct.cc

```diff
@@ -2358,9 +2358,9 @@ namespace SingleDuct {
 						if ( PltSizHeatNum > 0 ) {
 							CoilInTemp = TermUnitFinalZoneSizing( CurZoneEqNum ).DesHeatCoilInTempTU;
 							DesMassFlow = StdRhoAir * TermUnitSizing( CurZoneEqNum ).AirVolFlow;
-							DesZoneHeatLoad = CalcFinalZoneSizing( CurZoneEqNum ).DesHeatLoad * CalcFinalZoneSizing( CurZoneEqNum ).HeatSizingFactor;
```

Comment:
@rraustad Concerned this is going too deep.  The goal here was to get non-zero values in CalcFinalZoneSizing( CurZoneEqNum ).ZoneTempAtHeatPeak and CalcFinalZoneSizing( CurZoneEqNum ).ZoneHumRatAtHeatPeak when the design heating load is zero (and likewise for cooling).  It don't see why this should touch anything else, especially changing which sizing arrays are accessed.